.eslintrc.js

```diff
@@ -33,9 +33,12 @@ module.exports = {
     'promise/prefer-await-to-callbacks': 0,
     'promise/prefer-await-to-then': 0,
     'you-dont-need-lodash-underscore/get': 0,
+    'unicorn/prefer-spread': 0,
+    'unicorn/consistent-destructuring': 0,
+    'unicorn/no-array-push-push': 0,
```

Comment:
Should we add an issue to enable those rules?

Reply:
See https://github.com/netlify/cli/issues/1891